packages/react-reconciler/src/ReactFiberClassComponent.js

```diff
@@ -168,6 +171,11 @@ export default function(
       );
       stopPhaseTimer();
 
+      // Simulate an async bailout/interruption by invoking lifecycle twice.
+      if (invokePrecommitLifecycleHooksTwice) {
```

Comment:
What's the rationale for putting this one after the timer has ended?

Reply:
I think most of these are after the timer. Didn't seem important to put before or after, so long as it was outside of the timer. If you prefer I change it, I'm happy to.

Reply:
Nah it's fine either way I think, just curious